### PR TITLE
[RTC-186] Handle peer token

### PR DESCRIPTION
**Pull request by roznawsk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ room_id
 # => "8878cd13-99a6-40d6-8d7e-8da23d803dab"
 
 # Add peer
-{:ok, %Jellyfish.Peer{id: peer_id}} = Jellyfish.Room.add_peer(client, room_id, "webrtc")
+{:ok, %Jellyfish.Peer{id: peer_id}, token} = Jellyfish.Room.add_peer(client, room_id, "webrtc")
```
Comment (LVala) [inline README.md:39]: I'm not sure about that, but maybe `{:ok, %{peer: %Jellyfish.Peer{...}, token: token}}` would be better?
Honestly, I'm still not sure about the way we return the token from the backend (token in the Peer object/struct like you did in the first place or the way it is now).
@mickel8 what do you think about that?

Comment (mickel8) [reply README.md:39]: We can start with returning token in a tuple. I don't see any use cases for remembering token in server SDK at the moment. Once we discover such scenarios we will modify this code

**Comment by roznawsk** [comment]
The work related to adding authorization for http requests has been moved to another PR, so @LVala @mickel8 confirm that I can merge this one.

**Review by LVala — APPROVED** [review]
👍

**Pull request merged**